provisioning/ubuntu-provision.sh

```diff
@@ -689,14 +690,14 @@ function main() {
   install_common_requirements
   setuser # Define user Jenkins before all (to allow installing stuff in its home dir)
   install_git_gitlfs
+  install_JA_requirements
```

Comment:
What is the reason of moving this function call?

Reply:
using jq within the shell script that check the jdk

Reply:
It's not needed anymore since you are ensuring that `jq` is installed in the JDK installation :)